src/web/event-notifications/event-notification-types/SlackNotificationForm.jsx

```diff
@@ -3,8 +3,8 @@ import PropTypes from 'prop-types';
 import cloneDeep from 'lodash/cloneDeep';
 import get from 'lodash/get';
 
-import { Input } from 'components/bootstrap';
 import { getValueFromInput } from 'util/FormsUtils';
+import { Input } from 'components/bootstrap';
```

Comment:
What's the reason for this change?

Reply:
i had to revert a few changes in my front end code related to the task enabling and disabling of the backlog message field. probably, it changed the order of imports.